jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/pom.xml

```diff
@@ -38,6 +38,7 @@
     <version.org.jboss.shrinkwrap.resolver>3.3.0</version.org.jboss.shrinkwrap.resolver>
     <version.org.apache.maven>3.9.3</version.org.apache.maven>
     <version.org.apache.maven.resolver>1.9.13</version.org.apache.maven.resolver>
+    <version.com.google.guava>20.0</version.com.google.guava>
```

Comment:
guava 20? this version should not be used (CVEs https://www.cvedetails.com/vulnerability-list/vendor_id-1224/product_id-52274/Google-Guava.html)

Reply:
let see if it works for 30 but the curren shrinkwrap needs some alignments.

Reply:
I updated with 33.1.0-jre, which is the latest one and it looks like working for me locally. Let's see in the PR checks.